src/null-safety/index.md

```diff
@@ -115,6 +115,16 @@ To resolve these issues, check for
 of any packages you installed from pub.dev, and [migrate](#migrate) all
 of your own source code to use sound null safety.
 
+As of late January 2023, Dart 3 alpha is available from the Dart dev channel
+and the Flutter master channel, and we recommend you test your code
+for Dart 3 compatibility using that release:
+
+```terminal
+$ dart --version                     # make sure this reports 3.0.0-151.0.dev or higher
```

Comment:
I'm wondering, if `dart --version` _doesn't_ report that version back, or I already know I _don't_ have the dart dev channel or flutter master channel set up, how can I fix that? Maybe the steps are described elsewhere on this page, I didn't check.

But just in case, can we anchor link on "the Dart dev channel"/"the Flutter master" channel to instructions for setting up those channels, even if it's just a couple lines somewhere? And/or where it says "make sure" `dart --version` reports <...>, point them to what they need to do to get it to report that.

Reply:
Good point. I'm fixing that by adding instructions on the download page:
https://github.com/dart-lang/site-www/pull/4548